aws/cloudformation/s3_buckets.yml

```diff
@@ -20,6 +20,35 @@ Resources:
         BlockPublicPolicy: true
         IgnorePublicAcls: true
         RestrictPublicBuckets: true
+  DroneLibraryBucketPolicy:
```

Comment:
@uponthesun Suresh and I added a new bucket policy for the `cdo-v3-libraries` bucket in CI -- can you take a look?

Reply:
we just ran this changeset in CloudFormation, and it looks like there's an error in the policy we added. here's the event log from CloudFormation:

<img width="1010" alt="Screen Shot 2020-06-18 at 3 06 04 PM" src="https://user-images.githubusercontent.com/9812299/85076601-47ac8500-b175-11ea-9428-f6acb5e7fa92.png">